cirq-core/cirq/ops/measurement_gate_test.py

```diff
@@ -18,6 +18,15 @@
 import cirq
 
 
+def test_eval_repr():
+    # Basic safeguard against repr-inequality.
+    op = cirq.GateOperation(
+        gate=cirq.MeasurementGate(1, cirq.MeasurementKey(path=(), name='q0_1_0'), ()),
+        qubits=[cirq.GridQubit(0, 1)],
+    )
+    assert op == eval(repr(op))
```

Comment:
`cirq.testing.assert_equivalent_repr` won't work here?

Reply:
Good point - updated to use that instead.

Reply:
@vtomole, PTAL - merge is blocked on your review.